shared/js/media/video_player.js

```diff
@@ -105,19 +105,24 @@ function VideoPlayer(container) {
   var orientation = 0; // current player orientation
 
   // These are the raw (unrotated) size of the poster image, which
-  // must have the same size as the video.
+  // may have the same size as the video. We may find when we actually
+  // load the video they are different and resize the video element.
   var videowidth, videoheight;
+  var posterwidth, posterheight;
 
   var playbackTime;
   var capturedFrame;
 
-  this.load = function(video, posterimage, width, height, rotate, timestamp) {
+  this.load = function(video, posterimage, width, height, rotate, timestamp)
+  {
```

Comment:
nit: this `{` should probably remain on the previous line like it was to maintain consistency